teamcity.build

```diff
@@ -13,7 +14,15 @@
 
 	<property name="build.number" value="${CCNetLabel}" if="${property::exists('CCNetLabel')}" />
 
-	<target name="clean-configure-test" depends="cleanall init copy-teamcity-configuration binaries test verify-test-results binaries-zip" />
+	<target name="clean-configure-test" depends="cleanall init copy-teamcity-configuration binaries start-db-service test stop-db-service verify-test-results binaries-zip" />
+
+	<target name="start-db-service" unless="${skip.db-service or not property::exists('db-service')}">
+		<servicecontroller action="Start" service="${db-service}" timeout="120000" />
```

Comment:
On my local setup, Oracle takes up to one minute to start. So I have put a timeout of two minutes. (The default is five seconds.)

This of course means slowing down the builds a bit, especially for databases slow to start and stop.

We may dodge this by keeping a short timeout along with using `failonerror="false"` to ignore the timeout, and moving the service start before the build in order to rely on the build time (a bit more than 3 minutes on the TeamCity agent) for having the database ready. Similarly we could do a short ignored timeout that on the "stop service" target too, relying on the delay between two builds for having the service actually stopped before an attempt to restart it is done. (The source copy step maybe gives enough delay.)

But that could give flaky failures, so I think it is better to just wait.